src/utils.js

```diff
@@ -61,8 +61,8 @@ const fetchUploadData = async (opts) => {
 };
 
 const upload = async (opts) => {
-  const { path } = opts;
-  const endpoint = 'https://asset.cml.dev';
+  const { path, domain } = opts;
+  const endpoint = `https://${domain}`;
```

Comment:
with the combo little release, I think we can cross that bridge later?

Reply:
domain is concise but url to trailing `/` or not?

Reply:
> _cross that bridge later_

And keep growing our backwards-compatible API technical debt backpack? 🤔 I'd rather keep it as generic as possible since the beginning. 🤷🏼‍♂️

Reply:
so change to just a full `url` then?

Reply:
I would recommend taking a full `url` instead, as it would allow for plain `http` (argh!) or explicit endpoints in paths other than the server root. 👍🏼

Reply:
@iterative/cml, what would you prefer?